### PR TITLE
Whew, this PR is spicy!

### PR DESCRIPTION
**Pull request by F-OS**
:cl: optional name here
add: The spiciness of food is now properly modeled for races.
/:cl:

[why]: Because i feel this increases immersion without causing any major penalties.

### CODE REVIEW

File: code/modules/food_and_drinks/food.dm [hunk 1]
```diff
@@ -19,6 +19,9 @@
 	if(last_check_time + 50 < world.time)
 		if(ishuman(M))
 			var/mob/living/carbon/human/H = M
+			if((H.skin_tone == "caucasian1" || H.skin_tone == "albino") && prob(10))
+				to_chat(H,"<span class='warning'>Whew, that food was spicy!</span>")
+				H.reagents.add_reagent("capsaicin", 3)
```
Comment (yoyobatty) [inline code/modules/food_and_drinks/food.dm]: racism

**Comment by Kagus** [comment]
I admit, I chuckled.

This does affect some strange things though, such as rice pudding, cakes, and cheese.

File: code/modules/food_and_drinks/food.dm (hunk 1 above)
Comment (F-OS) [reply code/modules/food_and_drinks/food.dm]: you can't be racist towards whites check your privilege

Comment (Poojawa) [reply code/modules/food_and_drinks/food.dm]: I'm Texan and this offends my spicy tolerance

**Comment by ninjanomnom** [comment]
A roundstart trait would make sense but this doesn't make much sense as it is.

**Comment by PKPenguin321** [comment]
had a friend once who said ketchup was too spicy unironically
👍

**Comment by F-OS** [comment]
Why are you downvoting me proper modelling of realistic traits is important in a roleplaying game

**Comment by yoyobatty** [comment]
Because skin colour has no correlation to tolerance of spicy foods.

**Pull request closed without merging**